server/src/main/java/org/cloudfoundry/identity/uaa/account/AccountsController.java

```diff
@@ -46,8 +46,9 @@ public String activationEmail(Model model,
                                   @RequestParam(value = "client_id", required = false) String clientId,
                                   @RequestParam(value = "redirect_uri", required = false) String redirectUri,
                                   HttpServletResponse response) {
-        if (!IdentityZoneHolder.get().getConfig().getLinks().getSelfService().isSelfServiceLinksEnabled()) {
-            return handleSelfServiceDisabled(model, response, "error_message_code", "self_service_disabled");
+        if (!IdentityZoneHolder.get().getConfig().getLinks().getSelfService().isSelfServiceCreateAccountEnabled()) {
```

Comment:
Create a variable for IdentityZoneHolder.get().getConfig().getLinks().getSelfService() for easier reading

Reply:
Done